sceptre/cli/describe.py

```diff
@@ -0,0 +1,151 @@
+import click
+
+from helpers import catch_exceptions, get_stack, get_stack_and_env, write
+
+
+@click.group(name="list")
+def list_group():
+    """
+    Commands for listing attributes of stacks.
+
+    """
+    pass
```

Comment:
Should the reason for this be documented? I'm uncertain.

Reply:
Leaving as cli group behaviours are documented with Click documentation